packages/fxa-settings/src/lib/cache.ts

```diff
@@ -1,27 +1,67 @@
-import { InMemoryCache, gql } from '@apollo/client';
+import { InMemoryCache, gql, makeVar } from '@apollo/client';
 import Storage from './storage';
 import { Email } from '../models';
 
 const storage = Storage.factory('localStorage');
 
-export function sessionToken(newToken?: string) {
+export interface OldSettingsData {
+  uid: hexstring;
```

Comment:
This might be a "my machine" problem but I'm getting complaints on using `hexstring` (`Cannot find name 'hexstring'`), as well as where we're using `Hash`.

Reply:
it ought to be in your `react-app-env.d.ts` file. in vscode you may need to `Cmd+Shift+P` - `Restart TS server`